graph-manager-docs/source/integrations.md

```diff
@@ -119,83 +119,92 @@ The triggers you set up are evaluated on a rolling five minute window. For examp
 
 ## Datadog
 
-The Apollo Datadog integration allows you to forward the performance metrics that are available to you in Graph Manager to Datadog, which is convenient for teams already relying on Datadog for their monitoring. Datadog additionally supports an advanced function API, allowing teams to create sophisticated graphs and alerts for their GraphQL metrics.
+You can forward Graph Manager performance metrics to your Datadog account. Datadog supports an advanced function API, which enables you to create sophisticated graphs and alerts for GraphQL metrics.
```

Comment:
It would be good to mention the name of the integration somewhere here (the Apollo Datadog integration) so that users know what to call it in support tickets.